python/paddle/inference/__init__.py

```diff
@@ -49,4 +50,5 @@
     'get_num_bytes_of_data_type',
     'PredictorPool',
     'XpuConfig',
+    'InternalUtils',
```

Comment:
if `InternalUtils` is not public API, then delete this line and does not affect normal use.

Reply:
> if `InternalUtils` is not public API, then delete this line and does not affect normal use.

done